src/occa/internal/lang/modes/withLauncher.cpp

```diff
@@ -41,19 +43,24 @@ namespace occa {
         if (!success) return;
         setOklLoopIndices();
 
-        if (!success) return;
+        if (!success)
```

Comment:
Any reason for these changes?

Reply:
Apologies, this is probably some auto formatting again.